public/samples/DataStreams/StreamsUpkeep.sol

```diff
@@ -75,28 +63,28 @@ contract StreamsUpkeep is ILogAutomation, StreamsLookupCompatibleInterface {
         address quoteAddress;
     }
 
-    event ReportVerified(BasicReport indexed report);
-    event PriceUpdate(int192 price);
+    event PriceUpdate(int192 indexed price);
 
-    address public immutable FEE_ADDRESS;
+    IVerifierProxy public verifier;
+
+    address public FEE_ADDRESS;
     string public constant STRING_DATASTREAMS_FEEDLABEL = "feedIDs";
```

Comment:
these two storage variables dont need the STRING -- the type is already there in the statement

Reply:
Done